pipeline.py

```diff
@@ -192,6 +183,10 @@ def ilab_pipeline(
         output_model_registry_api_url=output_model_registry_api_url,
         output_model_name=output_model_name,
         output_model_version=output_model_version,
+        https_proxy_env_var_name=HTTPS_PROXY_ENV_VAR_NAME,
```

Comment:
Consider adding the https_proxy_env_var_name and no_proxy_env_var_name in consts.py to avoid adding these parameters

Reply:
These are well known environment variable names and won't change, so it's okay to hardcode them to avoid additional parameters in my opinion.